core/store/src/db.rs

```diff
@@ -104,7 +104,7 @@ pub enum DBCol {
     ColBlockOrdinal = 40,
     /// GC Count for each column
     ColGCCount = 41,
-    /// GC helper column to get all Outcome ids by Block Hash
+    /// All Outcome ids by Block Hash. For each block it is ordered by execution order.
     ColOutcomesByBlockHash = 42,
```

Comment:
I guess, we should reflect the change of the key type; it is no more the block hash alone, but also a shard id, right?